docs/design/colocation/Overview.md

```diff
@@ -0,0 +1,456 @@
+## Motivation
+
+Volcano now supports unified scheduling of microservices and batch workloads, it's compatible with the default scheduling algorithms of kubernetes, and provides rich scheduling policies for microservices and batch jobs, so it is suitable for unified scheduling of different types of jobs in colocation scenarios, and improves resource utilization simultaneously.
+
+## Architecture
+
+Volcano agent supports cloud-native online and offline workloads colocation. It runs as a daemonset in the kubernetes cluster, and is committed to improving the resource utilization of the kubernetes cluster and meanwhile ensures the Qos of online workloads through CPU/Memory/Network resource isolation and suppression.
+
+![architecture](images/architecture.jpg)
+
+### Qos Model
+
+|              Qos              |                 Typical  Usage Scenario                  |
+| :---------------------------: | :------------------------------------------------------: |
+|     LC(Latency Critical)      | Exclusive CPU for latency critical core online workloads |
+| HLS(Highly Latency Sensitive) |       Extremely latency sensitive online workloads       |
+|     LS(Latency Sensitive)     |      Latency sensitive approximate online workloads      |
+|        BE(Best Effort)        |    Offline AI/Big Data workloads, tolerable evictions    |
+
+You can set different qos levels to identify workloads of different priorities, and there is no seperate `priorityClass` definition to be consistent with k8s.
+
+### Features
+
+- Unified scheduling for all kinds of workloads including microservices and batch jobs.
+- Qos based resource model for online and offline workloads.
+- CPU burst: Allow containers to temporarily exceed the CPU limit to avoid throttling at critical moments.
+- Dynamic resource oversubscription: Dynamically calculate the resources that can be oversold based on the real-time CPU/Memory utilization of the node, and oversold resources can be used by offline workloads.
+- Network bandwidth isolation：Supports ingress network bandwidth limitation of the entire machine to ensure network usage for online workloads.
+
+## Quick start
+
+### Prepare
+
+Follow the [installation guide](https://github.com/volcano-sh/volcano?tab=readme-ov-file#quick-start-guide) to install volcano first, and then install volcano agent via kubectl, this way is both available for x86_64 and arm64 architecture. 
+
+```shell
```

Comment:
Volcano can be installed using the helm command. Does Volcano-agent support adaptation?

Reply:
Yeah we will support helm installation when the next version is officially released.

Reply:
This is first pr for the feature. Sure, this new component will be supported by helm later and it can be enable or disable by user as well.